### PR TITLE
fix readme.md typos

### PR DESCRIPTION
**Pull request by gigatexal**
simply ran aspell against the README.md. Don't want anyone to discount what a good project this is for a handful of typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,17 +12,17 @@ gorqlite should be considered alpha until more testers share their experiences.
...
-* Timeout can be set on a per-Connection basis to accomodate those with far-flung empires.
+* Timeout can be set on a per-Connection basis to accommodate those with far-flung empires.
 * Use familiar database URL connection strings to connection, optionally including rqlite authentication and/or specific rqlite consistency levels.
 * Only a single node needs to be specified in the connection.  gorqlite will talk to it and figure out the rest of the cluster from its redirects and status API.
 * Support for several rqlite-specific operations:
   * `Leader()` and `Peers()` to examine the cluster.
   * `SetConsistencyLevel()` can be called at any time on a connection to change the consistency level for future operations.
   * `Timing` can be referenced on a per-result basis to retrieve the timings information for executed operations as float64, per the rqlite API. 
-* `Trace(io.Writer)`/`Trace(nil)` can be used to turn on/off debugging information on everything gorqlite does to a io.Writer of your choice.
```
Comment (otoolep) [inline README.md]: This is actually correct, it's `io.Writer`.

**Comment by otoolep** [comment]
Thanks, I need you to revert one change.

**Comment by gigatexal** [comment]
Will do — sorry about the overly aggressive change. I’m not at a computer right now but will be in a few hours.

**Comment by gigatexal** [comment]
Done

File: README.md (hunk 1 above)
Comment (gigatexal) [reply README.md]: Should be fixed now.

**Comment by otoolep** [comment]
Thanks for your contributions.

**Pull request merged**